man/subscription-manager.8

```diff
@@ -1190,16 +1202,8 @@ subscription-manager register --username=admin --password=secret
 .fi
 .RE
 
-.SS LISTING, ATTACHING, AND REMOVING SUBSCRIPTIONS FOR PRODUCTS
```

Comment:
I've broken up this section into two separate sections since some of the commands  are being deprecated. I did this hoping that it would improve readability and prevent mixing deprecated commands with non-deprecated commands.

To whomever reviews this, let me know if this change is unnecessary and I can revert it.